getting-started/templates/systemlink-secrets.yaml

```diff
@@ -276,17 +276,21 @@ dataframeservice:
 ##
 feedservice:
```

Comment:
add a comment in PR description stating why DFS is not updated

Reply:
Updated